recovery/POST-RESTART-WALKTHROUGH.md

```
@@ -0,0 +1,338 @@
+
+**Grafana/Prometheus:**
+```bash
+kubectl get pods -n logging
+```
+
+---
+
+## Phase 7: Final Verification
+
+### Step 7.1: Test Application Access
+- Grafana: https://grafana.ari-sela.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)